TEAMS.md

```diff
@@ -0,0 +1,37 @@
+
+# People
+
+The following people are members of at least one team.
+The format is: `{preferred name}, {affiliation} (@{github_username})`
+
+* Tom Bentley, IBM (@tombentley)
+* Robert Young, IBM (@robobario)
+* Sam Barker, IBM (@SamBarker)
+* Grace Grimwood, IBM (@gracegrimwood)
+* Keith Wall, IBM (@k-wall)
+* Francisco Vila, IBM (@franvila)
```

Comment:
we've also got Paul with write access and code ownership of `/docs/` via the `
DocumentationWriters` team